inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -1530,44 +1608,51 @@ export class AnimationCurveEditorComponent extends React.Component<
                       </text>
                       <line x1={f.value} y1='0' x2={f.value} y2='5%'></line>
 
+                      {f.value % this.state.fps === 0 && f.value !== 0 ? (
+                        <line
+                          x1={f.value}
+                          y1='-100'
+                          x2={f.value}
+                          y2='5%'
+                        ></line>
+                      ) : null}
+
                       {this.isCurrentFrame(f.label) ? (
                         <svg>
                           <line
                             x1={f.value}
                             y1='0'
                             x2={f.value}
-                            y2='40'
+                            y2='-100%'
                             style={{
-                              stroke: 'rgba(18, 80, 107, 0.26)',
-                              strokeWidth: 6,
+                              stroke: 'white',
+                              strokeWidth: 0.4,
                             }}
                           />
+                          <svg x={f.value} y='-1'>
+                            <circle
+                              className='svg-playhead'
+                              cx='0'
+                              cy='0'
+                              r='2%'
+                              fill='white'
+                            />
+                            <text
+                              x='-0.6%'

```

Comment:
How are these values calculated? where does -0.6% comes from? or (0.17 * scale) em?

Reply:
I think I'll update those eventually. Move those to the css file. This is just margin for inside text.

Reply:
my main concern is responsibility of the entire panel. -0.6% is a value i don't remember seeing anywhere in the inspector, so i wonder where the values come from - from design maybe?

Reply:
Well, this is basically for design purposes, but I'm checking all these values again.